web/packages/shared/components/Controls/MultiselectMenu.tsx

```diff
@@ -164,8 +164,7 @@ export const MultiselectMenu = <T extends string>({
             <>
               <CheckboxInput
                 type="checkbox"
-                // @ts-expect-error assigning ReactNode to checkbox name field
-                name={opt.label}
+                name={opt.value}
```

Comment:
Oh yeah, `opt.value` is much better and actually aligns with what this attribute is for – it completely escaped my head when I was writing the original comment.